spark2/src/main/java/org/apache/iceberg/spark/source/Writer.java

```diff
@@ -113,6 +116,10 @@
     long tableTargetFileSize = PropertyUtil.propertyAsLong(
         table.properties(), WRITE_TARGET_FILE_SIZE_BYTES, WRITE_TARGET_FILE_SIZE_BYTES_DEFAULT);
     this.targetFileSize = options.getLong("target-file-size-bytes", tableTargetFileSize);
+
+    boolean tablePartitionedFanoutEnabled = PropertyUtil.propertyAsBoolean(
```

Comment:
Q:  will we set this for a given table ?  In my option,  it's per job ?

Reply:
we need set this option for a given table, Because some tables require fanout.